### PR TITLE
temporarly pin jsonschema

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

**Comment by takluyver** [comment]
I'd only do it for requirements.txt now, so that Travis can pass. This will be fixed before we release. I'd also use a `!=2.5.0` modifier.

**Comment by Carreau** [comment]
Does subproject like notebook that `pip install -e` this will get the things from requirement.txt ?

**Comment by takluyver** [comment]
Oh, right, no. But actually, why is it necessary to add it to requirements.txt at all? Shouldn't just changing the setup.py work?

**Comment by Carreau** [comment]
Well if you don't touch requirement.txt this repo would fail, but I guess not.  Let's try.

File: setup.py [hunk 1]
```diff
@@ -90,7 +90,7 @@
 install_requires = setuptools_args['install_requires'] = [
     'ipython_genutils',
     'traitlets',
-    'jsonschema>=2.0',
+    'jsonschema>=2.0,!2.5.0',
```
Comment (takluyver) [inline setup.py]: `!=`

Comment (Carreau) [inline setup.py]: Yup, saw that.

**Comment by Carreau** [comment]
Travis is happy. I was disturbed by the clock.

**Comment by aaren** [comment]
Does nbformat still need to be pinned to `jsonschema != 2.5.0`? Can this be relaxed now?

**Comment by minrk** [comment]
@aaren it's not pinned, it's something the opposite of pinned. There is one known-bad versions that shouldn't be allowed. If you install nbformat now, it will pull in jsonschema-2.5.1, which should be okay. I think we can leave this as-is, to protect against anyone somehow ending up with the broken 2.5.0 for a while.

**Comment by aaren** [comment]
Ah OK. There was an upgrade path I was following leading to a jsonschema / functools32 issue that had to be manually upgraded out of. I hadn't realised that 2.5.1 was OK.

Packaging is hard.

**Comment by minrk** [comment]
Super hard! Yes, I think 2.5.1 is okay.

**Pull request merged**